Detections/PulseSecureVPN/MultipleFailedVPNUserLogins.yaml

```diff
@@ -0,0 +1,26 @@
+id: 34663177-8abf-4db1-b0a4-5683ab273f44
+name: Multiple Failed VPN User Logins
+description: |
+  'Creates an incident in the event of multiple unsuccessful attempts to log into the VPN server, which could indicate a potential brute force attack.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes: 
+      - PulseSecureVPN
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess 
+relevantTechniques:
+  - T1110
+query: |
+
+  let timeframe = ago(1h);
+  PulseSecureVPN
+  | where TimeGenerated >= timeframe
+  | where Messages contains "Login failed"
+  | summarize count() by User, Source_IP
+  | where count_ > 20
+  | extend AccountCustomEntity = User, IPCustomEntity = Source_IP
```

Comment:
For use as a Detection, we will need to bring the Time value thru that we created in the summarize. In this case -
| extend timestamp = StartTime, AccountCustomEntity = User, IPCustomEntity = Source_IP

Reply:
Corrections made and pushed